pynicotine/slskproto.py

```diff
@@ -558,7 +558,8 @@ def bind_listen_port(self):
                 self.listen_socket.bind((ip_address, listenport))
                 self.listen_socket.listen()
                 self.listenport = listenport
-                log.add(_("Listening on port %i"), listenport)
+                log.add(_("Listening on port: %i"), listenport)
+                log.add(_("Maximum data sockets: %i"), MAXSOCKETS)
```

Comment:
In my opinion, this is still a difficult string for translators.

@hboetes what do you think?

Reply:
> It's a log entry, didn't you plan on not translating log entries?

Log entries outside the [Debug] categories are still translated.

On second thought, I'll just move this to the "Miscellaneous" debug category.